docs/en/faq.md

```diff
@@ -20,7 +20,27 @@ It may have been caused by the following reasons.
 pip install lmdeploy[all]
 ```
 
-2. If you have installed it and still encounter this issue, it is probably because you are executing turbomind-related command in the root directory of lmdeploy source code. Switching to another directory will fix it
+If you want to install the nightly build of LMDeploy's whl package, you can download and install it from the latest release at https://github.com/zhyncs/lmdeploy-build according to your CUDA and Python versions. Currently the update frequency of whl is once a day.
+
+2. If you have installed it and still encounter this issue, it is probably because you are executing turbomind-related command in the root directory of lmdeploy source code. Switching to another directory will fix it.
+
+But if you are a developer, you often need to develop and compile locally. The efficiency of installing whl every time is too low. You can specify the path of lib after compilation through symbolic links.
+
+```shell
+# mkdir and build locally
+mkdir bld && cd bld && bash ../generate.sh && ninja $(nproc)
+
+# go to the lmdeploy subdirectory from bld and set symbolic links
+cd ../lmdeploy && ln -s ../bld/lib .
```

Comment:
Will `ninja $(nproc)` install the so file to `../lmdeploy` folder ? If not, there may not have `../lmdeploy` folder

Reply:
> Will `ninja $(nproc)` install the so file to `../lmdeploy` folder ? If not, there may not have `../lmdeploy` folder

In the root directory of `lmdeploy`, there is a subdirectory named `lmdeploy` https://github.com/InternLM/lmdeploy/tree/main/lmdeploy

Reply:
lmdeploy (root directory) -> bld (build directory) -> lmdeploy (sub directory)

In the subdirectory named `lmdeploy`, `lib` is a **symbolic link** to the `lib` folder under `bld`.

```
lmdeploy
├── bld
│   ├── bin
│   ├── build.ninja
│   ├── cmake
│   ├── CMakeCache.txt
│   ├── CMakeFiles
│   ├── cmake_install.cmake
│   ├── compile_commands.json
│   ├── CPackConfig.cmake
│   ├── CPackSourceConfig.cmake
│   ├── _deps
│   ├── examples
│   ├── lib
│   ├── src
│   ├── tests
│   ├── TurboMindConfig.cmake
│   └── TurboMindTargets.cmake
└── lmdeploy
    ├── api.py
    ├── archs.py
    ├── cli
    ├── __init__.py
    ├── legacy
    ├── lib -> ../bld/lib
    ├── lite
    ├── __main__.py
    ├── messages.py
    ├── model.py
    ├── __pycache__
    ├── pytorch
    ├── serve
    ├── tokenizer.py
    ├── turbomind
    ├── utils.py
    ├── version.py
    └── vl
```